src/InstrumentationEngine.Lib/ConfigurationLoader.cpp

```diff
@@ -219,6 +219,7 @@ HRESULT CConfigurationLoaderHelper::ProcessInstrumentationMethodNode(_In_ BSTR b
 #else
 
 #include <string.h>
+#include <iconv.h>
```

Comment:
Would you mind adding an issue for this, and attaching the full error that you get from your compiler, including the compiler version?

Note, we don't currently have any validation running for mac machines.